Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Script_file_added_to_startup-related_Registry_keys.yml

```diff
@@ -0,0 +1,2980 @@
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "39":
+    id: "39"
+    taskid: 6a9dfdda-5aa5-449a-966e-d51c9f7c371f
+    type: regular
+    task:
+      id: 6a9dfdda-5aa5-449a-966e-d51c9f7c371f
+      version: -1
+      name: Check the processes reputation
+      description: Checks the reputation of the CGO, actor, and action processes using the specified hash.
+      script: '|||file'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      file:
+        complex:
+          root: alert
+          accessor: cgosha256
+          transformers:
+          - operator: append
+            args:
+              item:
+                value:
+                  simple: alert.osparentsha256
+                iscontext: true
+          - operator: uniq
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 433,
+          "y": 335
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "43":
+    id: "43"
+    taskid: 357da12c-5fcc-4eec-8dbf-ff8f712069e4
+    type: title
+    task:
+      id: 357da12c-5fcc-4eec-8dbf-ff8f712069e4
+      version: -1
+      name: Analysis
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "53"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 433,
+          "y": -50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "50":
+    id: "50"
+    taskid: f8da1674-a3c2-46b6-8aa9-75628b0e6795
+    type: title
+    task:
+      id: f8da1674-a3c2-46b6-8aa9-75628b0e6795
+      version: -1
+      name: Script File Analysis
+      tags:
+      - prevalence_check
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "104"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1128,
+          "y": 1052
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "53":
+    id: "53"
+    taskid: ef3721b8-94c4-4bec-9f66-056906400434
+    type: regular
+    task:
+      id: ef3721b8-94c4-4bec-9f66-056906400434
+      version: -1
+      name: Get registry data
+      description: Returns additional alert data, including registry keys and values that reference the script file.
```

Comment:
i don't know if 'alert data' here is referring to xsiam alerts, but if so, the same issue as described above with alerts in 2.x and issues in 3.x